src/components/course/course-header/data/hooks/useRedemptionStatus.js

```diff
@@ -17,6 +17,8 @@ const useRedemptionStatus = () => {
 
   const handleRedeemSuccess = (transaction) => {
     queryClient.invalidateQueries({ queryKey: ['policy'] });
+    queryClient.invalidateQueries({ queryKey: ['redeemablePolicies'] });
```

Comment:
[curious/suggestion] I wonder if there's a reasonable way to make the relevant `useQuery` hooks for these above query keys rely on a similar query key structure, given they're all related to policies. That way, we'd only need a single `invalidateQueries` call to invalidate all policy-related queries, e.g.:

```jsx
export const enterpriseUserSubsidyQueryKeys = {
  // Namespace for all user subsidy query keys
  all: ['user-subsidy'],
  policy: [...enterpriseUserSubsidyQueryKeys.all, 'policy'],
  // Used with query against `can-redeem` API endpoint
  coursePolicyRedeemability: ({
    enterpriseId, lmsUserId, courseRunKeys, activeCourseRunKey,
  }) => [
    ...enterpriseUserSubsidyQueryKeys.policy,
    enterpriseId,
    'can-redeem',
    { lmsUserId, courseRunKeys, activeCourseRunKey },
  ],
  // Used with query to fetch user's redeemable subsidy access policies
  redeemablePolicies: (enterpriseId, userId) => [...enterpriseUserSubsidyQueryKeys.policy, 'redeemable-policies', enterpriseId, userId],
  // Used with query for polling pending policy transactions after initial policy redemption
  pollPendingPolicyTransaction: (transaction) => [
    ...enterpriseUserSubsidyQueryKeys.policy,
    'transaction',
    transaction,
  ],
};
```

Then, the following would invalidate all the `policy`-associated queries above.

```jsx
queryClient.invalidateQueries({ queryKey: enterpriseUserSubsidyQueryKeys.policy });
```

Reply:
Refactored successfully, tests seem to be passing as expected, and the local enrollment experience went flawlessly after the refactor 👍🏽